freqtrade/strategy/interface.py

```diff
@@ -338,6 +338,32 @@ def informative_pairs(self) -> ListPairsWithTimeframes:
 # END - Intended to be overridden by strategy
 ###
 
+    def gather_informative_pairs(self) -> ListPairsWithTimeframes:
+        """
+        Internal method which gathers all informative pairs (user or automatically defined).
+        """
+        informative_pairs = self.informative_pairs()
+        for attr_name in dir(self.__class__):
+            cls_attr = getattr(self.__class__, attr_name)
+            if not callable(cls_attr):
+                continue
+            if not getattr(cls_attr, '_is_informative', False):
+                continue
+            timeframe = getattr(cls_attr, '_timeframe', None)
+            asset = getattr(cls_attr, '_asset', None)
+            assert asset is not None
+            assert timeframe is not None
+            if asset:
+                if '/' in asset:
+                    pair = asset
+                else:
```

Comment:
I would not start supporting this - as it'll fail in the future and represent a breaking change (and provides limited benefits at this time, too).

While currently we only have spot pairs - this is likely going to change.
Once you venture out of spot pairs - you'll have pairs which ain't seperated by "/" (BTC-PERP on FTX, for example).
As such - relying on "/" as seperator will not work - and would result in `BTC-PERP/USD` - which however is not a valid pair - and is therefore a very nasty bug to debug.

Reply:
You are right. This is bit of a pickle though. Specifying full pair in a decorator is complicated, because we may switch between USD/USDT/USDC/BUSD stake currencies and that would require strategy modification. There is no way to look up stake currency in config as at the time decorator is executed we have no access to config. Do you have any idea if we can somehow avoid hardcoding stake currency part, or do we have to live with this limitation?

Reply:
maybe allow the shorthand of `"{stake}"` too ... ? ;) as i'm usually pretty type-lazy :laughing: